src/main-client.ts

```diff
@@ -310,6 +315,36 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/account/apiRestrictions');
   }
 
+  acceptQuoteRequest(
+    params: EnableConvertSubAccountParams,
+  ): Promise<any>; {
+    return this.postPrivate('sapi/v1/convert/acceptQuote', params);
+  }
```

Comment:
Typo here is breaking the code, should be:
```suggestion
  acceptQuoteRequest(
    params: EnableConvertSubAccountParams,
  ): Promise<any> {
    return this.postPrivate('sapi/v1/convert/acceptQuote', params);
  }
```